kubeai/models/llama-3.1-8b-instruct-cpu.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Source: models/templates/models.yaml
+apiVersion: kubeai.org/v1
+kind: Model
+metadata:
+  name: llama-3.1-8b-instruct-cpu
+spec:
+  features: [TextGeneration]
+  url: hf://meta-llama/Meta-Llama-3.1-8B-Instruct
+  cacheProfile: standard
```

Comment:
cache profile -- why nfs for one and standard for another?  Do we need to explain to the user what to use when.

Reply:
These are examples of use. The user can choose which storage solution to use. Admins can ad more. We can add more details in follow-up PRs.